.github/workflows/ci.yml

```
@@ -0,0 +1,32 @@
+    
+    - name: Install dependencies
+      run: npm ci
+    
+    - name: Run ESLint
+      run: npm run lint
+    
+    - name: Run TypeScript check
+      run: npm run typecheck
+    
+    - name: Run unit tests
+      run: npm test
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)